bundles/AdminBundle/public/js/pimcore/asset/image.js

```diff
@@ -427,9 +403,6 @@ pimcore.asset.image = Class.create(pimcore.asset.asset, {
 
             this.previewContainerId = 'pimcore_asset_image_preview_' + this.id;
             this.previewMode = 'image';
```

Comment:
```suggestion
```

Should be obsolete ...

Reply:
same for https://github.com/pimcore/pimcore/blob/b54d1a6a961f8a7e07de9a01deca1593b8a6a18d/bundles/AdminBundle/public/js/pimcore/asset/image.js#L435

Reply:
@Corepex thank you, these files are removed now, seems like they were indeed not needed anymore